boards/hail/src/main.rs

```diff
@@ -315,7 +315,8 @@ pub unsafe fn reset_handler() {
     let button = components::button::ButtonComponent::new(board_kernel).finalize(
         components::button_component_helper!((
             &sam4l::gpio::PA[16],
-            capsules::button::GpioMode::LowWhenPressed
+            capsules::button::GpioMode::LowWhenPressed,
+            kernel::hil::gpio::FloatingState::PullUp
```

Comment:
```suggestion
            kernel::hil::gpio::FloatingState::PullNone
```
Hail has a pull up resistor on board.